### ISSUE TITLE
[a11y design review]: Allergies domain

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#55626]
## Auditor(s)

Sara Smith

## Design Review
Based on the development of the prototype, some of the following may or may not be applicable. 
- [ ] [Color and contrast checks](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#use-of-color)
  - [ ] WCAG 2.1 AA
  - [ ] WCAG 2.1 AAA
  - [ ] [Who Can Use](https://www.whocanuse.com/?b=ecece5&c=000000&f=20&s=)
  - [ ] High Contrast Mode
- [ ] [Keyboard navigation](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#keyboard-nav)
  - [ ] Annotations for Tab order, reflow and reading order (if needed)
- [ ] Accessible Name Annotations
  - [ ] [Alt attributes for images](https://webaim.org/techniques/alttext/)
  - [ ] Text alternative for complex images such as charts graphs maps etc.
- [ ] Language (ease of reading)
- [ ] HTML Tags
  - [ ] [Page Title](https://accessibility.18f.gov/page-titles/)
  - [ ] [Heading order](https://accessibility.18f.gov/headings/)
   - [ ] Annotations (if needed)
  - [ ] Semantic Consideration for paragraphs ordered lists numbering
  - [ ] [Tables (heading rows)](https://design.va.gov/components/table)
- [ ] Size of clickable areas

### As Time Allows
- [ ] [Components](https://design.va.gov/components/) follow VA design system guidance.
- [ ] [Patterns](https://design.va.gov/patterns/) follow VA design system guidance.

## Prototype Link

 - In Sketch, on the "For Devs and Content" page: 
    - List view [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/ADEEDDEE-A648-49F6-A50A-2627A6C3F517) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/8BA5C800-0E78-4F56-BB79-E3E323DCAA13)
    - Details view [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/E4EB5CBF-0056-49C9-828B-C57C30E8D948) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/BB92EA3D-DBB7-4991-B930-D3CA3841C27E)
    - [Alerts](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/BB92EA3D-DBB7-4991-B930-D3CA3841C27E)

### Concise issue description
- Where you found it
- More details

#### Bandaid solution
- If helpful, provide a temporary solution that involves less content/design changes

#### Recommended solution
- Try to provide an ideal solution too that explores alternatives should content, design, IA, the broader service, etc can be fixed
- If no research has been conducted with assistive technology users, it should be a priority

### Concise issue description
- Where you found it
- More details

#### Bandaid solution
- If helpful, provide a temporary solution that involves less content/design changes

#### Recommended solution
- Try to provide an ideal solution too that explores alternatives should content, design, IA, the broader service, etc can be fixed
- When possible, point to more research with assistive technology users

### Concise issue description
- Where you found it
- More details

#### Bandaid solution
- If helpful, provide a temporary solution that involves less content/design changes

#### Recommended solution
- Try to provide an ideal solution too that explores alternatives should content, design, IA, the broader service, etc can be fixed
- When possible, point to more research with assistive technology users

**Comment by sara-amanda** [comment]
This was consolidated for the Medical Records Midpoint Review.
